src/components/mypage/PostSection/PostSection.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PostList from '@components/home/PostList/PostList';
+import * as S from './PostSection.Styled';
+
+interface PostSectionProps {
+  posts: any[];
+  handleBookmarkClick: (postId: number) => void;
+}
+
+const PostSection: React.FC<PostSectionProps> = ({ posts, handleBookmarkClick }) => (
+  <S.PostSection>
+    <S.PostInfo>
+      <S.PostText>게시물</S.PostText>
+      <S.PostCount>6개</S.PostCount>
+    </S.PostInfo>
+    <PostList
+      posts={posts}
+      handleBookmarkClick={handleBookmarkClick}
+      $isMypage={window.location.pathname === '/mypage'}
```

Comment:
```suggestion
      const location = useLocation();
      ....
      $isMypage={location.pathname === '/mypage'}
```
`window`객체 사용하면 컴포넌트가 URL 변경 시, 감지하지 못해서 새로고침 해야될 가능성이 생겨서 `useLocation` 사용하는게 더 좋을 것 같아요!